test/e2e/predictor/test_torchserve.py

```diff
@@ -58,3 +58,34 @@ def test_torchserve_kserve():
     res = predict(service_name, "./data/torchserve_input.json")
     assert(res.get("predictions")[0] == 2)
     kserve_client.delete(service_name, KSERVE_TEST_NAMESPACE)
+
+
+def test_torchserve_v2_kserve():
```

Comment:
Can we add the gRPC test as well?

Reply:
This requires proto files. Shall i upload it to data folder along with json input files.

Reply:
alternatively you can write a gRPC python client